input/pagecontent/changes.md

```diff
@@ -7,6 +7,14 @@ This change log documents the significant updates and resolutions implemented fr
 
 #### Changes in this version
 - New profiles: 
+<<<<<<< HEAD
```

Comment:
Hi @johnscarter, this and other files in this PR have merge conflicts that need to be resolved. Please resolve and send back to me for review. Thank you!

Reply:
@johnscarter, the PR isn't quite ready is ready for review, as it is missing updates to the profiles notes section that we agreed to workshop first. 

I suggest closing this PR and opening a new one once all issues are resolved and required changes are applied.